user_manual/groupware/calendar.rst

```diff
@@ -225,7 +225,7 @@ You can also tell when the recurrence ends.
 Trash bin
 ~~~~~~~~~
 
-If you delete events, tasks or a calendar in Calendar, your data won't be gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period of 30 days those items will be deleted permanently. You can also permanently delete earlier if you wish.
+If you delete events, tasks or a calendar in Calendar, your data won't be gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period which defaults to 30 days (your administrator may have changed this setting), those items will be deleted permanently. You can also permanently delete earlier if you wish.
```

Comment:
administrator -> administration

Reply:
your data won't be gone yet. -> your data is not gone yet.

Reply:
You can also permanently delete earlier if you wish -> You can also permanently delete items earlier if you wish

Reply:
To facilitate changes 😉.
```suggestion
If you delete events, tasks or a calendar in Calendar, your data is not gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period which defaults to 30 days (your administration may have changed this setting), those items will be deleted permanently. You can also permanently delete items earlier if you wish.
```

Reply:
To facilitate changes 😉.
```suggestion
If you delete events, tasks or a calendar in Calendar, your data is not gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period which defaults to 30 days (your administration may have changed this setting), those items will be deleted permanently. You can also permanently delete items earlier if you wish.
```

Reply:
> administrator -> administration

used just like that in other docs as well. this is a bigger, separate change, isn't it?

is there a glossary with terms to use / not to use?

Reply:
Administrator is a person (can be one or more).
Administration is a team of administrators. Is the more official wording.

Reply:
> used just like that in other docs as well. this is a bigger, separate change, isn't it?

Yes, this is a bigger change. But IMHO this is an ongoing change. We have many occurences of "admin" and "administrator". "admin" is an abbreviation and if we use "administration" we get gender neutral.

Valdnet and me correcting all new strings. Mostly in nextcloud projects and second in docs.
Please use it.